### ISSUE TITLE
Prevent Page error 504 when a node is offline

### ISSUE THREAD

**Issue by ZEROPOINTBRUH** [pelican-dev/panel#170]
Okay, this is annoying for the few that are using pterodactyl and have to strictly use cloudflare proxies or anything else to proxy through in that matter. When a server goes offline or is not responding, we just want a solution to where it shows the server is offline instead of tossing a 504 please and thank you.

**Comment by iamkubi** [comment]
Agree, this is a problem.  I looked into this previously and found that Laravel and React use these error statuses to communicate, so the laravel controller that checks the wings status uses a 504 error to indicate to React that that node is offline.  Cloudflare (in my opinion rightfully) sees a 504 error being returned and throws up an error page, however it seems like the order(?) the nodes respond in matters.

If you have 2 nodes and 1 node is down you will get the cloudflare error 50% of the time.
If you have 3 nodes and 1 node is down you will get the cloudflare error 33% of the time.

I think this can be fixed by using different status codes for this communication, or to communicate with something other than error states.

**Comment by notAreYouScared** [comment]
This should have been fixed with https://github.com/pelican-dev/panel/commit/111b8b3cda2bf08c48d30fd96a51bcf9e7ff9ba2

@ZEROPOINTBRUH Can you verify? I use CF as a proxy and CF throws a 502 when wings is offline, and the panel sees that as a null and error.
